waterlog/src/Components/TankComponent.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { fetchTankLevelById } from '../actions/TankLevelsByIdAction';
+
+export class TankComponent extends Component {
+  componentWillMount() {
+    this.props.fetchTankLevelById(this.props.id);
+  }
+
+  getTankImage = (percent) => {
+    if (percent === 100) {
+      return <img src="/images/100_tank.png"
+        width="150"
```

Comment:
Your styles are inline, please extract it to a css file and make use of a css class instead

Reply:
done